src/cli/commands/add.js

```diff
@@ -216,6 +207,7 @@ export function hasWrapper(commander: Object): boolean {
 
 export function setFlags(commander: Object) {
   commander.usage('add [packages ...] [flags]');
+  commander.option('-W, --ignore-workspace-root-check', 'required to run yarn add inside a workspace root');
```

Comment:
Interesting short-hand flag name.

Reply:
Didn't think long about this one, but you make me remember it conflicts with `-W(arning)` of compilers 😞 oh well, should be fine for this iteration.